src/dlgTriggerEditor.cpp

```diff
@@ -3594,6 +3616,11 @@ void dlgTriggerEditor::saveAction()
                 pItem->setIcon( 0, iconError );
                 pItem->setText( 0, name );
             }
+
+            // If not active, don't bother raising the TToolBar for this save.
+            if( !pT->isActive() ) {
```

Comment:
Script errors do cause some problems.  I should change this so toolbars with errors don't get displayed, that is how it works for buttons right now but the code to show and hide the toolbar widget was never there so odd things happen when `pT->state()` is `false`.